src/nimUtils.ts

```diff
@@ -209,6 +209,7 @@ export function getBinPath(tool: string): string {
     if (process.env['PATH']) {
         // add support for choosenim
         process.env['PATH'] = process.env['PATH'] + (<any>path).delimiter + process.env['HOME'] + '/.nimble/bin';
+        process.env['PATH'] = process.env['PATH'] + (<any>path).delimiter + process.env['USERPROFILE'] + '/.nimble/bin';
```

Comment:
I dont sure that will work for non windows environment

Reply:
My new code works just as well in a non-Windows environment as the old code did in a Windows environment: which is to say, it causes no harm. For both lines of code, if the environment variable isn't already defined, it just ends up adding the root `/.nimble/bin` directory to the PATH. A check for whether the variable is defined is possible, but I chose to change as little as possible from the original.

Reply:
I recommend to update PR by changes like this:
```
if (process.platform === 'win32') {
   process.env['PATH'] = process.env['PATH'] + (<any>path).delimiter + process.env['USERPROFILE'] + '/.nimble/bin';
} else {
   process.env['PATH'] = process.env['PATH'] + (<any>path).delimiter + process.env['HOME'] + '/.nimble/bin';
}
```